### ISSUE TITLE
Bug RESTORE fail

### ISSUE THREAD

**Issue by lookxx** [vanowm/FirefoxCookiesManagerPlus#64]
Restore all options and restore selected stopped working with this update

Restore ALL no work

**Comment by vanowm** [comment]
Any specifics? It fails how?

**Comment by Sbubbolio** [comment]
I have the same problem:

1) right click, select "restore all"
2) select previously backupped-cookies-txt-file **
3) instead of adding the cookies to the list it does nothing

...and if you drag-and-drop the file in the cookiemanager-form it says "No cookies restored" **

 *\* I tried with old-version-backup-file and with new-version-backup-file too

Please fix it :)
Thank you

Firefox version: 46.0.1
CookieManager+ version: 1.7.1

**Comment by Sbubbolio** [comment]
I am adding the logs (of this operation: "right click, restore all, selecting the backup file, close CM+"):

```
CM+ (cookiesmanagerplus.js:563:1): (debug):[click] cookieClickEvent() execTime: 260434
CM+ (export.js:534:1): (string):#Created by Cookies Manager+ v1.7.1 on Tue May 10 2016 09:44:43 GMT+0200 (ora solare Europa occidentale)



#template:{HOST}    {ISDOMAIN_RAW}  {PATH}  {ISSECURE_RAW}  {EXPIRES_RAW}   {NAME}  {CONTENT_RAW}       {TYPE_RAW}





.google.com TRUE    /   FALSE   0   NID 79=P6lqg5wlp4lCUpNYILFeyNqrIA_znRFUp7NhASlEPy0NswzVrla7FDOzewyqfzdCm8fCWDqGQoWs_WOdJTEClGfjlUjdKSPetEyEMjYMYUJIKrIfrHNwLUy0yODFop5H     1

.youtube.com    TRUE    /   FALSE   0   CONSENT WP.2533de       1

.youtube.com    TRUE    /   FALSE   0   PREF    f1=50000000&f5=30030        1

.youtube.com    TRUE    /   FALSE   0   VISITOR_INFO1_LIVE  gCo8dQ0uSuk     1

.youtube.com    TRUE    /   FALSE   0   YSC PY_po0Jij9A     1
CM+ (cookiesmanagerplus.js:3235:1): (debug):menuView() execTime: 265282
CM+ (cookiesmanagerplus.js:1781:1): (debug):[canceling false] cookieSelected() execTime: 265283
CM+ (cookiesmanagerplus.js:671:1): (debug):selectionSave() execTime: 265286
CM+ (cookiesmanagerplus.js:721:1): (debug):[start save selection to file] selectionSave() execTime: 265287
CM+ (cookiesmanagerplus.js:1283:1): (debug):clearCookieProperties() execTime: 265288
CM+ (cookiesmanagerplus.js:1913:1): (debug):UI_EnableCookieBtns() execTime: 265292
CM+ (cookiesmanagerplus.js:2490:1): (debug):infoRowsSort() execTime: 265293
CM+ (cookiesmanagerplus.js:573:1): (debug):unload() execTime: 267976
CM+ (commonFunc.js:566:1): (debug):[protect observer removed] protect_CookieKeeper_unload() execTime: 267989
CM+ (cookiesmanagerplus.js:671:1): (debug):selectionSave() execTime: 267990
CM+ (cookiesmanagerplus.js:676:1): (debug):[building list] selectionSave() execTime: 267991
CM+ (cookiesmanagerplus.js:719:1): (debug):[end build] selectionSave() execTime: 1 [267992]
CM+ (cookiesmanagerplus.js:721:1): (debug):[start save selection to file] selectionSave() execTime: 267993
CM+ (cookiesmanagerplus.js:758:1): (debug):[end save selection to file] execute() execTime: 2 [267995]
CM+ (cookiesmanagerplus.js:630:1): (debug):unload() execTime: 267995
```

Maybe you could find it useful...

**Comment by Sbubbolio** [comment]
Ok, I noticed you already fixed it in in the 1.7.2b2 version
https://github.com/vanowm/FirefoxCookiesManagerPlus/blob/main/xpi/%7Bbb6bc1bb-f824-4702-90cd-35e2fb24f25d%7D_v1.7.2b2.xpi

To install it you must set "xpinstall.signatures.required" to false in firefox's "about:config"
(I'm writing this, so it can be useful to users like me... I discovered it only now)

... however it would be great to have a new version downloadable from firefox website, instead of changing that parameter ;)

**Comment by vanowm** [comment]
Unfortunately it still fails with encrypted backups, large encrypted backups that is.

**Comment by vanowm** [comment]
Should be fixed in v1.8
